prow/tide/tide_test.go

```diff
@@ -3453,6 +3453,21 @@ func TestAccumulateReturnsCorrectMissingTests(t *testing.T) {
 		{
 			name:       "Result from successful context gets respected",
 			presubmits: map[int][]config.Presubmit{1: {{Reporter: config.Reporter{Context: "job-1"}}}},
+			prs: []PullRequest{{
+				Number:     githubql.Int(1),
+				HeadRefOID: githubql.String("headsha"),
+				Commits: struct{ Nodes []struct{ Commit Commit } }{Nodes: []struct{ Commit Commit }{{Commit: Commit{
+					OID: githubql.String("headsha"),
+					Status: CommitStatus{Contexts: []Context{{
+						Context:     githubql.String("job-1"),
+						Description: githubql.String("Job succeeded. BaseSHA:" + baseSHA),
+						State:       githubql.StatusStateSuccess,
+					}}},
+				}}}}}},
+		},
+		{
+			name:       "Result from successful context gets respected with deprecated baseha delimiter",
+			presubmits: map[int][]config.Presubmit{1: {{Reporter: config.Reporter{Context: "job-1"}}}},
```

Comment:
This diff is confusing to read, the old delimiter is not shown because it was there before